test/forge/FeeTest.sol

```diff
@@ -348,4 +349,63 @@ contract FeeTest is IntegrationTest {
         assertEq(assets, expectedAssets, "assets");
         assertGe(assets, assetsBefore, "assets increased");
     }
+
+    function testUpdateWithdrawQueueRemovingDisabledMarketWithFee(
+        uint256 firstAmountSupplied,
+        uint256 secondAmountSupplied,
+        address newFeeRecipient,
+        uint256 blocks
+    ) public {
+        firstAmountSupplied = bound(firstAmountSupplied, MIN_TEST_ASSETS, MAX_TEST_ASSETS);
+        secondAmountSupplied = bound(secondAmountSupplied, MIN_TEST_ASSETS, MAX_TEST_ASSETS);
+        blocks = _boundBlocks(blocks);
+        vm.assume(newFeeRecipient != FEE_RECIPIENT && newFeeRecipient != address(0));
+
+        _setCap(allMarkets[0], firstAmountSupplied);
+        _setCap(allMarkets[1], secondAmountSupplied);
+
+        Id[] memory supplyQueue = new Id[](2);
+        supplyQueue[0] = allMarkets[0].id();
+        supplyQueue[1] = allMarkets[1].id();
```

Comment:
not necessary if we inverse the lines above

Reply:
Yes because i believe there's the idle market

Reply:
yes it's necessary or yes it's not?

Reply:
it's necessary